opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectHeader.jsx

```diff
@@ -485,7 +485,7 @@ const StixDomainObjectHeader = (props) => {
           )}
           <StixCoreObjectFileExport
             id={stixDomainObject.id}
-            type={stixDomainObject.entity_type}
+            type={entityType}
```

Comment:
This could never work as `stixDomainObject` is a relay object and `entity_type` is inside a fragment

The entity type is given as prop here, so we have it already.